src/continual/multimodal_trainer.py

```
@@ -0,0 +1,510 @@
+                add_special_tokens=True
+            )
+            input_ids = torch.tensor(
+                encoding.input_ids,
+                dtype=torch.long
+            )
+        else:
+            # Fall back to text-only
+            if text is None:
+                raise ValueError(
+                    "Text required when using text-only tokenizer"
+                )
```

Ruff (0.14.2)

Avoid specifying long messages outside the exception class

(TRY003)